test/new-e2e/utils/e2e/client/agent.go

```diff
@@ -30,6 +35,63 @@ func (agent *Agent) init(auth *Authentification, data *agent.ClientData) error {
 	return err
 }
 
-func (agent *Agent) Status() (string, error) {
-	return agent.sshClient.Execute("sudo datadog-agent status")
+func (agent *Agent) Version() (string, error) {
+	return agent.sshClient.Execute("datadog-agent version")
+}
+
+type Status struct {
+	Content string
+}
+
+func newStatus(s string) *Status {
+	return &Status{Content: s}
+}
+
+// isReady true if status contains a valid version
+func (s *Status) isReady() (bool, error) {
+	return regexp.MatchString("={15}\nAgent \\(v7\\.\\d{2}\\..*\n={15}", s.Content)
```

Comment:
The return code should be enough, otherwise we probably need to change something in the Agent code.